logistics/scripts/modules/tools/ToolsHelper.psm1

```diff
@@ -195,6 +195,46 @@ function Test-DotNetCore {
     }
 }
 
+# TO DO: split into windows and unix install functions
+# document the need for unzip and wget
+function Install-SqlPackage {
```

Comment:
Can we remove this change? This func isn't called

Reply:
removed the function